packages/vscode-container-client/src/contracts/ContainerClient.ts

```diff
@@ -1708,48 +1404,20 @@ export type ListFilesCommandOptions = CommonCommandOptions & {
     operatingSystem?: ContainerOS;
 };
 
-export type ListFilesItem = {
-    /**
-     * The name of the file/directory
-     */
-    name: string;
-    /**
-     * The absolute path of the file/directory within the container
-     */
-    path: string;
-    /**
-     * The size of the file (0 if a directory), in bytes
-     */
-    size: number;
-    /**
-     * The type of the file item (file/directory)
-     */
-    type: FileType;
-    /**
-     * The mode (permissions) of the file in base 10
-     */
-    mode?: number;
-    /**
-     * The (container) uid of the user the file belongs to
-     */
-    uid?: number;
-    /**
-     * The (container) gid of the user the file belongs to
-     */
-    gid?: number;
-    /**
-     * The modification time of the file/directory, in milliseconds since Unix epoch
-     */
-    mtime?: number;
-    /**
-     * The creation time of the file/directory, in milliseconds since Unix epoch
-     */
-    ctime?: number;
-    /**
-     * The access time of the file/directory, in milliseconds since Unix epoch
-     */
-    atime?: number;
-};
+export const ListFilesItemSchema = z.object({
+    name: z.string().describe('The name of the file/directory'),
+    path: z.string().describe('The absolute path of the file/directory within the container'),
+    size: z.number().describe('The size of the file (0 if a directory), in bytes'),
+    type: z.nativeEnum(FileType).describe('The type of the file item (file/directory)'),
+    mode: z.number().optional().describe('The mode (permissions) of the file in base 10'),
+    uid: z.number().optional().describe('The (container) uid of the user the file belongs to'),
+    gid: z.number().optional().describe('The (container) gid of the user the file belongs to'),
+    mtime: z.number().optional().describe('The modification time of the file/directory, in milliseconds since Unix epoch'),
+    ctime: z.number().optional().describe('The creation time of the file/directory, in milliseconds since Unix epoch'),
```

Comment:
I found out recently that ctime is not actually creation time, it's "changed" time. ctime marks when file metadata was changed (such as permissions etc.) OR the file content was changed, whereas mtime marks only when file content was changed.